types/react-table/index.d.ts

```diff
@@ -362,22 +362,22 @@ export interface ComponentProps {
 
 export interface TextProps {
     /** Default: 'Previous' */
-    previousText: string;
+    previousText: React.ReactNode | React.ComponentType;
```

Comment:
`normalizeComponent` is not called for `previousText`, hence it can only be a `ReactNode`:
https://github.com/react-tools/react-table/blob/master/src/pagination.js#L80